.dockerignore

```diff
@@ -0,0 +1,14 @@
+.git
+.project
+.vscode
+.settings
+.vs_code
+.settings
+.eslintrc*
+tsconfig*
+yarn.lock
+apollo.config.js
```

Comment:
Should apollo.config be ignored in the container?

Reply:
yah, don't need to pass anything into the container once the bundles are built.